packages/bpk-component-spinner/src/BpkExtraLargeSpinner.tsx

```diff
@@ -31,12 +29,16 @@ import STYLES from './BpkSpinner.module.scss';
 const getClassName = cssModules(STYLES);
 
 type Props = {
-  type: SpinnerTypes,
-  className?: string,
+  type?: SpinnerTypes;
+  className?: string | null;
```

Comment:
I'm wondering if we need semantic difference between null and undefined here?

Reply:
Possibly? I was jsut setting it here based on what we had as the previous right now to avoid any unexpected change or issues